dir.targets

```diff
@@ -57,6 +57,12 @@
       <DestinationDir>test-runtime/</DestinationDir>
     </SupplementalTestData>
   </ItemGroup>
+  
+  <PropertyGroup>
```

Comment:
I think we wanted to make this change specific to the repo that it was going into in case their were any properties that we wanted to be custom to it.